src/sass/layout/header.scss

```diff
@@ -0,0 +1,33 @@
+header {
+    height: 100vh;
+}
+.nav {
+background-image: url("../image/bg-nav-header.jpg");
+background-position: center;
+background-repeat: no-repeat;
+background-size: cover;
+text-align: center;
+&-item {
+    margin-top: 10px;
+}
+&-link {
+    text-decoration: none;
+    color: $color-third;
+    text-transform: uppercase;
+    font-weight: bold;
+
+}
+}
+
+@media (min-width: 768px){
+    .nav-item{
+        font-size: 20px;
+        padding-top: 25px;
+}
+}
+
+@media (min-width: 640px) and (orientation:landscape){
```

Comment:
hmmm to media query jest napisane zle, postaraj sie uzyc breakpointow z pliku simple-css-layout